Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM circleci/node:6.10 
+FROM circleci/node:8.11 
```

Comment:
Has this been tested somehow?

Reply:
yeah, all the circleci tests on the cumulus repo, use this image.